server/src/main/java/com/linecorp/centraldogma/server/metadata/MetadataService.java

```diff
@@ -999,6 +999,53 @@ public CompletableFuture<Revision> deactivateToken(Author author, String appId)
                                       }));
     }
 
+    /**
+     * Update the {@link Token} of the specified {@code appId} to user.
+     */
+    public CompletableFuture<Revision> updateTokenToUser(Author author, String appId) {
+        requireNonNull(author, "author");
+        requireNonNull(appId, "appId");
+
+        return tokenRepo.push(INTERNAL_PROJECT_DOGMA, Project.REPO_DOGMA, author,
+                              "Update the token: " + appId + " to user",
+                              () -> tokenRepo
+                                      .fetch(INTERNAL_PROJECT_DOGMA, Project.REPO_DOGMA, TOKEN_JSON)
+                                      .thenApply(tokens -> {
+                                          final JsonPointer adminPath = JsonPointer.compile(
+                                                  "/appIds" + encodeSegment(appId) +
+                                                  "/admin");
+                                          final Change<JsonNode> change = Change.ofJsonPatch(
+                                                  TOKEN_JSON,
+                                                  new ReplaceOperation(adminPath,
+                                                                       Jackson.valueToTree(
+                                                                               false)).toJsonNode());
+                                          return HolderWithRevision.of(change, tokens.revision());
+                                      }));
+    }
+
+    /**
+     * Update the {@link Token} of the specified {@code appId} to admin.
+     */
+    public CompletableFuture<Revision> updateTokenToAdmin(Author author, String appId) {
```

Comment:
I think we can merge these two methods with a boolean parameter. Could you merge them?

Reply:
Okay, it could be more simple.